packages/core/src/ceramic.ts

```diff
@@ -500,10 +500,11 @@ export class Ceramic implements StreamReaderWriter, StreamStateLoader {
    */
   async _init(doPeerDiscovery: boolean): Promise<void> {
     try {
-      this._logger.imp(
-        `Connecting to ceramic network '${this._networkOptions.name}' using pubsub topic '${this._networkOptions.pubsubTopic}'`
-      )
-
+      if (!EnvironmentUtils.useRustCeramic()) {
```

Comment:
not sure if we want to keep logging this in all cases or change how we try to drive/force the bootstrapping from js-ceramic when running in remote mode

Reply:
I definitely think we should still log what network we are on.  So maybe if we're using ceramic-one we log a slightly different message that says the network and says that we are using Recon for data synchronization or something like that.

Reply:
I log a message but it could be wrong if the c1 node is remote as it's running whatever network it started on.. I could probably figure out a way to pipe it all the way down but I don't see that we know or care normally, since the ipfs node is always remote from our perspective here.